rest/snippets/UserRecordings/RecordingEntity.yml

```diff
@@ -110,6 +110,76 @@ RecordingEntity:
                   example: "7bb76a23-29de-49ba-a862-9d4e18fe1328"
                   description: The ID of the related resource, UUID.
 
+UngroupedRecordingEntity:
```

Comment:
Ungrouped recordings follow the exact same schema as grouped ones. The auto_recording_request and series relationships are still required, but they will just be null in this case. You should use the same entity.

Reply:
Update that entity though and set `nullable: true` on the `data` property for those relationships. Also add a description like with `auto_recording_request` explaining when those can be null.